crates/starknet/src/starknet/state_update.rs

```diff
@@ -96,20 +94,23 @@ mod tests {
 
         // execute the same transaction, but increment nonce, so new transaction hash could be
         // computed
-        declare_txn.common.nonce = Felt::from(1);
-        let (txn_hash, _) = starknet.add_declare_transaction_v2(declare_txn).unwrap();
-        let tx = starknet.transactions.get_by_hash_mut(&txn_hash).unwrap();
-        assert_eq!(tx.finality_status, Some(TransactionFinalityStatus::AcceptedOnL2));
-        assert_eq!(tx.execution_result.status(), TransactionExecutionStatus::Succeeded);
-
-        let state_update = starknet
-            .block_state_update(starknet_rs_core::types::BlockId::Tag(
-                starknet_rs_core::types::BlockTag::Latest,
-            ))
-            .unwrap();
-
-        assert!(state_update.declared_classes.is_empty());
-        assert!(state_update.cairo_0_declared_classes.is_empty());
+        // TODO: uncomment when migrated to blockifier
```

Comment:
"uncomment when migrated to blockifier" so uncomment now or later? is it addressed in another task?

Reply:
removed in https://github.com/0xSpaceShard/starknet-devnet-rs/commit/01a0bfa2631159f354b921eb57283fe650aa7d71